Makefile

```diff
@@ -2,31 +2,39 @@
 
 help:
 	@echo "clean-build - remove build artifacts"
-	@echo "clean-pyc - remove Python file artifacts"
-	@echo "build - build so file from pyx"
+	@echo "clean-test - remove Python file artifacts"
+	@echo "clean-eggs - remove cached eggs"
+	@echo "build - build package"
 	@echo "lint - check style with flake8"
 	@echo "test - run tests quickly with the default Python"
 	@echo "test-all - run tests on every Python version with tox"
-	@echo "coverage - check code coverage quickly with the default Python"
 	@echo "docs - generate Sphinx HTML documentation, including API docs"
 	@echo "release - package and upload a release"
 	@echo "dist - package"
 
-clean: clean-build clean-pyc
+clean: clean-build clean-test clean-eggs
 	rm -rf htmlcov/
 
 clean-build:
 	rm -rf build/
 	rm -rf dist/
 	rm -rf *.egg-info
 
-clean-pyc:
-	find . -name '*.pyc' -exec rm -f {} +
-	find . -name '*.pyo' -exec rm -f {} +
-	find . -name '*~' -exec rm -f {} +
+.PHONY: clean-test
+clean-test:
+	rm -rf __pycache__/
+	rm -rf .pytest_cache/
```

Comment:
Given the following hypothetical directory structure:

```
$ tree -a .
.
├── __pycache__
│   └── file
├── .pytest_cache
│   └── file
└── test
    ├── __pycache__
    │   └── file
    ├── .pytest_cache
    │   └── file
    └── test.py
```

the commands:

```
rm -rf __pycache__/
rm -rf .pytest_cache/
```

will only delete the top-level directories:

```
.
└── test
    ├── __pycache__
    │   └── file
    ├── .pytest_cache
    │   └── file
    └── test.py
```

because `__pycache__` folders may be created for any modules in the `test` directory, wouldn't the intention here be to delete any `__pycache__` or `.pytest_cache` directory recursively?